guides/07_FAQs.md

```diff
@@ -59,6 +59,12 @@ Note that due to limitations in transpilers, requirements of the web platform mi
 
 [Android Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=635686&sort=-stars&q=android%20chrome%20bluetooth%20headphone&can=2) has a known issue switching between bluetooth audio output devices. While clients can continue the meeting using the default device, there is a [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=635686&sort=-stars&q=android%20chrome%20bluetooth%20headphone&can=2) related to switching to an bluetooth audio output.
 
+### I am getting `Cannot select audio output device. This browser does not support setSinkId` error on the browser console. Is this a known issue?
+
+In the background, `bindAudioElement()`, `bindAudioStream()` and `bindAudioDevice()` call the browser API [`setSinkId()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/setSinkId). The full list of browsers that support `setSinkId` API is given here https://caniuse.com/?search=setsinkid. In Firefox, this feature is behind the `media.setsinkid.enabled` preference (needs to be set to `true`). To change preferences in Firefox, visit `about:config`
+
+If you wish to find whether the browser supports `setSinkId()`, you can use the `supportsSetSinkId()` helper function in BrowserBehavior. 
```

Comment:
"Use `BrowserBehavior.supportsSetSinkId()` to determine whether the browser supports `setSinkId()` before calling these methods."

Reply:
Fixed